kubetest2/internal/deployers/eksapi/deployer.go

```diff
@@ -186,6 +194,11 @@ func (d *deployer) Up() error {
 	}
 	if d.deployerOptions.StaticClusterName != "" {
 		klog.Infof("inited k8sclient, skip the rest resource creation for static cluster")
+		if err := d.DeployBusyboxAndWaitForNodes(); err != nil {
```

Comment:
Why do we need these?

Reply:
the e2e test suite has a before suite that requires certain nodes to be ready before deploy any testing pods. So deployment this to get nodeclaim work